addons/sys_gui/fnc_volumeKeyDown.sqf

```diff
@@ -15,7 +15,23 @@
  * Public: No
  */
 
-if (!alive acre_player || dialog || ACRE_IS_SPECTATOR || GVAR(volumeOpen)) exitWith {false};
+if (!alive acre_player || ACRE_IS_SPECTATOR || GVAR(volumeOpen)) exitWith {false};
+
+// Abort on open Dialogs that aren't an ACE Progressbar
+private _aceProgressBar = displayNull;
+if (
+    dialog && 
+    {
```

Comment:
The next check is a boolean, so lazy eval can't be useful here.

Reply:
True that